guide/payments/payment-request-formats.md

```diff
@@ -0,0 +1,147 @@
+
+**LNURL-Withdraw**
+
+Allows for invoices to offer withdrawals with optional minimum and maximum amounts.
+
+**LNURL-Auth**
+
+Lets users authenticate to third-party services via their wallet (Lightning node).
+
+**LNURL-Channel**
+
+Streamlines the channel opening user flow.
+
+### [Lightning addresses](https://lightningaddress.com)
+
+This identifier builds on top of LNURL-Pay and introduces the familiar format of email addresses as identifiers.
+
+## Examples
+
+Each request format has its own unique approach to bundle payment information for transmission. Unique are Lightning addresses, which are easily readable and memorizable by users.
+
+{% include fact/open.html color="blue" label="On-chain address" icon="triangle" %}
+
+<span style="word-break: break-word;">1E4NmSEoT8WhBfSo4fBCUS3NLXM4u8txLF</span>
+
+{% include fact/close.html %}
+
+{% include fact/open.html color="blue" label="Lightning invoice (BOLT 11)" icon="triangle" %}
+
+<span style="word-break: break-word;">lnbc1pvjluezsp5zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zygspp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdpl2pkx2ctnv5sxxmmwwd5kgetjypeh2ursdae8g6twvus8g6rfwvs8qun0dfjkxaq9qrsgq357wnc5r2ueh7ck6q93dj32dlqnls087fxdwk8qakdyafkq3yap9us6v52vjjsrvywa6rt52cm9r9zqt8r2t7mlcwspyetp5h2tztugp9lfyql</span>
+
+{% include fact/close.html %}
+
+{% include fact/open.html color="blue" label="Offer (BOLT 12)" icon="triangle" %}
+
+<span style="word-break: break-word;">lno1pg257enxv4ezqcneype82um50ynhxgrwdajx283qfwdpl28qqmc78ymlvhmxcsywdk5wrjnj36jryg488qwlrnzyjczs</span>
+
+{% include fact/close.html %}
+
+{% include fact/open.html color="blue" label="Lightning AMP invoice" icon="triangle" %}
+
+<span style="word-break: break-word;">lnbc1pvjluezsp5zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zyg3zygspp5qqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqqqsyqcyq5rqwzqfqypqdpl2pkx2ctnv5sxxmmwwd5kgetjypeh2ursdae8g6twvus8g6rfwvs8qun0dfjkxaq9qrsgq357wnc5r2ueh7ck6q93dj32dlqnls087fxdwk8qakdyafkq3yap9us6v52vjjsrvywa6rt52cm9r9zqt8r2t7mlcwspyetp5h2tztugp9lfyql</span>
+
+{% include fact/close.html %}
+
+{% include fact/open.html color="blue" label="Lightning node ID" icon="triangle" %}
+
+<span style="word-break: break-word;">026e6215d95fa7ef6c3d214ffa24be987dec4aa2d9474b82a6dd07c0d927d3640b@35.234.119.46:9735</span>
+
+{% include fact/close.html %}
+
+{% include fact/open.html color="blue" label="LNURL-Pay" icon="triangle" %}
+
+<span style="word-break: break-word;">LNURL1DP68GURN8GHJ7MRWW4EXCTT5DAHKCCN00QHXGET8WFJK2UM0VEAX2UN09E3K7MF0W5LHZ0TPVC6RQENRXQMXXDFNXSCRYDE3V5UXXDNXVDNX2DNYXGURXER98QURWWP3XAJNZVPHVGCXYDPCVDJRSC3S8QUNWVF5XVUNSVF4XG6NGAKYX3T</span>
+
+{% include fact/close.html %}
+
+{% include fact/open.html color="blue" label="LNURL-Withdraw" icon="triangle" %}
+
+<span style="word-break: break-word;">LNURL1DP68GURN8GHJ7MRWW4EXCTT5DAHKCCN00QHXGET8WFJK2UM0VEAX2UN09E3K7MF0W5LHZ0FHXSMRSVNXVESKVEPEX9NRGETYX5ENGVTYXCUR2ENYVE3KVDFKXYCXVDF4XCURSVR9XFJR2DMPX9NRVETPXPSKVCTPX3SKGEPCXGCRYKNQXCW</span>
```

Comment:
These don't have to be capitalized though they should be - could make a note on that on this page also.

Reply:
Thanks. I wasn't sure on that.

Reply:
Capitalized these.